README.md

```diff
@@ -10,30 +9,22 @@ Minetest is a free open-source voxel game engine with easy modding and game crea
 Copyright (C) 2010-2022 Perttu Ahola <celeron55@gmail.com>
 and contributors (see source file comments and the version control log)
 
-Table of Contents
-------------------
-
-1. [Further Documentation](#further-documentation)
-2. [Default Controls](#default-controls)
-3. [Paths](#paths)
-4. [Configuration File](#configuration-file)
-5. [Command-line Options](#command-line-options)
-6. [Compiling](#compiling)
-7. [Docker](#docker)
-8. [Version Scheme](#version-scheme)
-
-
-Further documentation
-----------------------
-- Website: https://minetest.net/
-- Wiki: https://wiki.minetest.net/
-- Developer wiki: https://dev.minetest.net/
-- Forum: https://forum.minetest.net/
-- GitHub: https://github.com/minetest/minetest/
-- [doc/](doc/) directory of source distribution
-
-Default controls
-----------------
+## In Case You Downloaded the Source Code
+
+If you downloaded the Minetest Engine source code in which this file is
+contained, you probably want to download the [Minetest Game](https://github.com/minetest/minetest_game/)
```

Comment:
merge blemish

Reply:
What?

Reply:
sorry, I meant that this got mistakenly left in when merge conflicts were resolved, it doesn't exist in the master branch anymore

Reply:
Ok, will remove